dump/src/util/dump/Dump.java

```
@@ -1060,6 +1067,10 @@ void writeDictionary() throws IOException {
    }
 
    void writeMeta() throws IOException {
+      if ( isReadonly() ) {
+         return;
```

This should probably throw an exception?

---

I guess so, yeah... Let's throw.

---

Done.